app/helpers/manage_jwt.js

```diff
@@ -7,3 +7,12 @@ exports.generate = (data, expInSeconds) => {
 
   return jwt.encode({ exp, iat, data }, config.tokenSecret);
 };
+
+exports.decode = token => jwt.decode(token, config.tokenSecret);
+
+exports.getTokenFromReq = req => {
+  const { authorization } = req.headers;
+  const token = authorization.replace('Bearer ', '');
+
+  return exports.decode(token);
```

Comment:
`decode()` doesn't check the token's validity, you should verify. Also, after you have decoded the payload from the token you should add it to the `req` object, something like `req.authUser = tokenPayload`